libs/langchain/langchain/retrievers/multi_query.py

```diff
@@ -43,10 +43,14 @@ def parse(self, text: str) -> LineList:
 )
 
 
+def _unique_documents(documents: Sequence[Document]) -> List[Document]:
+    return [doc for i, doc in enumerate(documents) if doc not in documents[:i]]
```

Comment:
This looks a bit odd, should we compute a hash?

Reply:
then we have to worry about non-hashable (or hard-to-hash) metadata